File: JavaTpoint_Programs/Java/BasicPrograms/FibonacciSeries.java
public class FibonacciSeries {
    public static void main(String[] args) {
        int n = 10, a = 0, b = 1;
        System.out.print("Fibonacci Series: ");
        for (int i = 1; i <= n; i++) {
            System.out.print(a + " ");
            int sum = a + b;
            a = b;
            b = sum;
        }
    }
}

// FibonacciSeries is the series in which the previous code is the sum of old 2 numbers

File: JavaTpoint_Programs/Javascript/BasicPrograms/FibonacciSeries.js
let n = 10

let a = 0, b = 1;

// for (let i = 0; i < n; i++) {
//   console.log(a);
//   [a, b] = [b, a + b];
// }

for (let i = 1; i <= n; i++) 
{
    console.log(a + " ")
    let sum = a + b;
    a = b;
    b = sum;
}
